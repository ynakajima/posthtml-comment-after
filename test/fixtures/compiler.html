<html>
  <body>
    <div id="wrapper" class="container">
      <p class="wow foooo">OMG</p>
    </div>
  </body>
</html>
