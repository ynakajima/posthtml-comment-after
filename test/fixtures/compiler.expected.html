<html>
  <body>
    <div id="wrapper" class="container">
      <p class="wow foooo">OMG</p><!-- ðŸ‘ˆ This Element has .wow !!! -->
    </div>
  </body>
</html>
